src/Pages/TemplateSelect.js

```diff
@@ -1,42 +1,142 @@
 import { Link } from 'react-router-dom';
 import { TemplateCard } from '../components/TemplateCard';
-import styled from 'styled-components';
+// import styled from 'styled-components';
```

Comment:
Let`s avoid having comments of unused stuff

Reply:
will do